### ISSUE TITLE
Keybinding in french layout

### ISSUE THREAD

**Issue by XL64** [microsoft/WSL#11041]
### Windows Version

Microsoft Windows [version 10.0.22631.3007]

### WSL Version

2.0.9.0

### Are you using WSL 1 or WSL 2?

- [x] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.133.1-microsoft-standard-WSL2

### Distro Version

Ubuntu 22.04

### Other Software

When trying to undo things (Ctrl + underscore) inside emacs -nw I get the delete behaviour (previous char is deleted).
shokey show also that the ket send is not C-_ but ^?:
```
 showkey -a

Press any keys - Ctrl-D will terminate this program

^?      127 0177 0x7f
^D        4 0004 0x04
```

### Repro Steps

Run wsl with french keyboard layout
Start emacs -nw
go to scratch buffer (C-x b return)
write some text
move cursor in the middle of the text
hit C-_ 
the char before prompt is deleted instead of last written one

### Expected Behavior

C-_ should correspond to undo

### Actual Behavior

C-_ correspond to delete

**Comment by XL64** [comment]
Have seen another issue with indian layout that was adresses to windows terminal but that occurs with all terminal windows native I tested (https://github.com/microsoft/terminal/issues/11194).
It does not happen with emacs graphical or ubuntu graphical terminal (terminator installed via apt-get)

**Comment by XL64** [comment]
Any news/comment on that ?

**Comment by OneBlue** [comment]
This issue is related to emacs configuration. WSL doesn't have control over emacs' behavior

**Comment by XL64** [comment]
No it's not. shokeys -a show the same issue with no link to emacs configuration

**Comment by XL64** [comment]
Sorry for that it seems to be a Terminal issue which was solved in 1.20.
